extensions/configuration-editing/src/settingsDocumentHelper.ts

```diff
@@ -192,6 +197,18 @@ export class SettingsDocument {
 		return Promise.resolve([]);
 	}
 
+	private provideDefaultLanguageCompletionItems(location: Location, range: vscode.Range): vscode.ProviderResult<vscode.CompletionItem[]> {
```

Comment:
Can we make this method reused between the two clients calling `vscode.languages.getLanguages` here, it seems there is quite a bit of overlap